sqlmesh/core/model/kind.py

```diff
@@ -104,15 +104,19 @@ def model_kind_name(self) -> t.Optional[ModelKindName]:
         return self
 
 
-def _unique_key_validator(v: t.Any) -> t.List[str]:
-    if isinstance(v, exp.Identifier):
-        return [v.name]
-    if isinstance(v, (exp.Tuple, exp.Array)):
-        return [e.name for e in v.expressions]
-    return [i.name if isinstance(i, exp.Identifier) else str(i) for i in v]
+def _unique_key_validator(
```

Comment:
Shouldn't we reuse this implementation: https://github.com/TobikoData/sqlmesh/blob/main/sqlmesh/core/model/meta.py#L139? It looks a lot more comprehensive

Reply:
done